package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "heroku": "^7.60.2"
+  }
```

Comment:
Was there an issue with heroku that required this package?  It should be able to be deployed without a new `package.json` file.  If this is removed then I would also delete the new `yarn.lock` file that was created